Makefile

```diff
@@ -28,8 +28,17 @@ test:  ## Run all tests
 test-unit:  ## Run unit tests
 	go test ./...
 
-test-e2e:  ## Run end-to-end tests
-	cd e2e && docker compose build && docker compose run test && docker compose down --remove-orphans
+# Run Cypress locally against dockerised dev server
+# make sure the dockerised dev server is running first with `make dev`
+test-e2e-dev:
+	cd web && CYPRESS_baseUrl=http://localhost:8000 npx cypress run --headless --browser chrome
+
+## Run cypress against the release build
+test-e2e-release:
+	cd e2e && \
+	docker compose -p "ucl-arc-tre-e2e" build && \
```

Comment:
probably worth extracting this into a var cf. https://github.com/ucl-arc-tre/portal/blob/cc1f4055164ee528605cddaae9ec4476ce43969c/Makefile#L2

Reply:
https://github.com/ucl-arc-tre/portal/pull/33/commits/d99a7b070c491d43a0ce9b8b9263f40f543e3398